omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PatientIdentifierTypeController1_8Test.java

```diff
@@ -168,4 +168,21 @@ public void shouldReturnTheAuditInfoForTheFullRepresentation() throws Exception
 		
 		assertNotNull(PropertyUtils.getProperty(result, "auditInfo"));
 	}
+	
+	@Test
+	public void shouldReturnCustomRepresentationForAuditInfo() throws Exception {
```

Comment:
Shouldn't this be in `ConversionUtilTest`?

Reply:
`PatientIdentifierTypeController1_8Test` also include a test named `shouldReturnTheAuditInfoForTheFullRepresentation`. So it's nice to have a matching E2E test for converting to a custom representation.

The new method that we introuduced in `ConversionUtil` is `private`. So I think it makes sense to have it here instead.

Reply:
Are you directly calling this private method from the test?

Reply:
No, the private method is not called directly in the test. The test is to makes GET calls to the resource with custom representation as the `v` param.

https://github.com/openmrs/openmrs-module-webservices.rest/blob/9eb0e4d0c8250580c5c2a30c5326b480cc6b9fdc/omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PatientIdentifierTypeController1_8Test.java#L174-L176

Assertions in this test should be enough to prevent someone from breaking things again :)

Reply:
If the test is for a change in ConversionUtil, is there a reason why you do not put the test in ConversionUtilTest?

Reply:
@gayanW did you see the above comment?

Reply:
Sorry for the late reply. I've been busy with work.

I don't think it would be good to put Mock Http test in ConversionUtil.

Additionally I could write a new unit test for `ConversionUtil#convertToRepresentation` in `ConversionUtilTest`. Let me check that on the weekend.

Reply:
I added a new unit test to `ConversionUtilTest`.

Reply:
@dkayiwa Can you check :)